.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Build
+      run: dotnet build /p:Configuration=Release
+
+    - name: Upload
+      uses: actions/upload-artifact@v3
+      with:
+        name: CraftingRevisions
+        path: ./CraftingRevisions/bin/Release/net6.0/CraftingRevisions.dll
+        if-no-files-found: error
```

Comment:
Down below I mentioned Nuget generation. It would be ideal if the built nuget package also got uploaded. Just duplicate this step, assign new names, and correct the path.